### PR TITLE
Ap google oauth

### PR DESCRIPTION
**Pull request by aplasket**
PR Template:

# high level what you did:
- add google oauth and link to login with google

## Details:
- added google oauth gem
- added sessions controller with a create and destroy functionality
- added current user
- added a basic index view to redirect the login to after signing in using google

## Questions or Notes:
note that more testing needs to be done for this to satisfy user story 1. there is some light functionallity such as destroy. but that is not currently tested

## Overall Checklist:
-[x] tests written for all methods added
-[x] test coverage above 90%
-[x] RSpec running clean
-[x] ran rails s -- features working properly!
-[] pulled/merged milestone branch before submitting PR

### CODE REVIEW

**Review by jcjurado3 — COMMENTED** [review]
Great work on getting Google OAuth to work!

File: Gemfile [hunk 1]
```diff
@@ -67,3 +71,15 @@ group :development do
...
+  gem "capybara"
+  gem "launchy"
+  gem "simplecov"
+  gem "vcr"
+  gem "webmock"
+  gem "shoulda-matchers"
+  gem "orderly"
+end
+
+gem 'omniauth-google-oauth2'
```
Comment (JCBeldo) [inline Gemfile:85]: That's the gem!

**Review by Jredish11 — COMMENTED** [review]
nice work yall

**Review by JCBeldo — APPROVED** [review]
Nice way to get the oauth to work. :) Glad you figured out the path being the issue.

**Pull request merged**